src/queries/dash/domain-list.sql

```diff
@@ -0,0 +1,93 @@
+--- description: List of domains along with some summary data.
+--- Access-Control-Allow-Origin: *
+--- Cache-Control: max-age=3600
+--- timezone: UTC
+--- device: all
+--- domainkey: secret
+
+WITH pvs AS (
+  SELECT
+    weight,
+    REGEXP_REPLACE(hostname, r'^www.', '') AS hostname,
+    COUNT(DISTINCT id) AS ids,
+    FORMAT_DATE('%Y-%b', time) AS month,
+    MIN(time) AS first_visit,
+    MAX(time) AS last_visit
+  FROM
+    helix_rum.EVENTS_V3(
+      '-',
+      -1,
+      -1,
+      '2020-01-01',
+      '2099-12-31',
```

Comment:
I'm okay to make url a parameter but in practice it will never be used as the purpose of this query is to show all domains.  For date, what I can do is change the dashboard by consolidating total (lifetime) visits and current month visits into one column which is last 60 days visits to make the query less expensive.  This will marginally change the resultset -- specifically https://github.com/adobe/helix-run-query/blob/cc1fef26a877f9c182c88844a1054c11a8b3646a/src/queries/dash/domain-list.sql#L91-L92 will have to change to something like
`a.recent_visits >= 1000`
which means legit low-traffic or seasonal sites (such as rockstar.adobeevents.com) would be excluded.  Please confirm before I move forward on this change.

Reply:
Then make the default interval 365 days (we don't have more data anyway), this will at least allow us to continue using it for the next 76 years.

Reply:
This is done.  We lose go-live date for sites older than a year but that is okay.  I will change the UI accordingly to set expectations.